apps/OboeTester/app/src/main/java/com/mobileer/oboetester/DynamicWorkloadActivity.java

```diff
@@ -283,25 +259,26 @@ public void onClick(View view) {
             mAffinityBoxes.add(checkBox);
             checkBox.setText(cpuIndex + "");
             checkBox.setOnClickListener(checkBoxListener);
-            if (cpuIndex == defaultCpuAffinity) {
+            if (((1 << cpuIndex) & defaultCpuAffinityMask) != 0) {
                 checkBox.setChecked(true);
             }
         }
-        NativeEngine.setCpuAffinityMask(1 << defaultCpuAffinity);
+        NativeEngine.setCpuAffinityMask(defaultCpuAffinityMask);
 
         mMultiLineChart = (MultiLineChart) findViewById(R.id.multiline_chart);
         mMaxCpuLoadTrace = mMultiLineChart.createTrace("CPU", Color.RED,
                 0.0f, 2.0f);
         mWorkloadTrace = mMultiLineChart.createTrace("Work", Color.BLUE,
-                0.0f, (float)WORKLOAD_MAX);
+                0.0f, (1.2f * WORKLOAD_HIGH_MAX));
```

Comment:
Why 1.2?

Reply:
So I can show the CPU going above the highest workload. Arbitrary number based on tuning.